### PR TITLE
Add the documentation to the new scale-to-zero-pod-retention-period flag

### PR DESCRIPTION
**Pull request by vagababov**
Add docs for the new CM flag for last pod retention period.

/assign @abrennan89 @markusthoemmes

### CODE REVIEW

File: docs/serving/configuring-autoscaling.md [hunk 1]
```diff
@@ -526,6 +526,43 @@ spec:
 {{< /tab >}}
 {{< /tabs >}}
 
+
+### Scale To Zero Last Pod Retention Period
+
+This flag ensures that after the decision has been made by the Autoscaler to scale to 0 the last pod will remain active for at least this amount of timei, i.e minimum amount of time the last pod will linger. This is in contrast to `scale-to-zero-grace-period` flag, that determines the maximum amount of time that the last pod will stay alive.
```
Comment (abrennan89) [inline docs/serving/configuring-autoscaling.md:532]: The `scale-to-zero-grace` flag determines the **minimum** amount of time in seconds that the least pod will remain active after the Autoscaler has decided to scale pods to zero.

**NOTE:** This contrasts with the `scale-to-zero-grace-period` flag, which determines the **maximum** amount of time in seconds that the least pod will remain active after the Autoscaler has decided to scale pods to zero.

File: docs/serving/configuring-autoscaling.md [hunk 2]
```diff
@@ -496,7 +496,7 @@ This period is an upper bound amount of time the system waits internally for the
 
 * **Global key:** `scale-to-zero-grace-period`
 * **Per-revision annotation key:** n/a
-* **Possible values:** Duration
+* **Possible values:** Duration, at least 6s.
```
Comment (abrennan89) [inline docs/serving/configuring-autoscaling.md:499]:
```suggestion
* **Possible values:** Duration (must be at least 6s).
```

File: docs/serving/configuring-autoscaling.md (hunk 1 above)
Comment (vagababov) [reply docs/serving/configuring-autoscaling.md]: Done.

File: docs/serving/configuring-autoscaling.md (hunk 2 above)
Comment (vagababov) [reply docs/serving/configuring-autoscaling.md]: Done.

**Pull request merged**